server/models/UserModel/index.js

```diff
@@ -58,6 +58,25 @@ const UserModel = {
       .collection(collections.users)
       .updateOne({ uid }, { $set: userAttributes })
   },
+  findOne: async (db, uid) => {
+    return await db.collection(collections.users).findOne(
+      { uid },
+      {
+        projection: {
```

Comment:
Should this use the `userMongoProjection` for consistency?

Reply:
@taylorkearns The projection object instructs mongo that the query should omit or include only certain properties. When a property has the value of 1 it will be included in the result, if it's a value of 0 it will be omitted from the result.